src/DIRAC/WorkloadManagementSystem/Service/PilotManagerHandler.py

```diff
@@ -77,9 +74,9 @@ def export_getCurrentPilotCounters(cls, attrDict={}):
     types_addPilotReferences = [list, str]
 
     @classmethod
-    def export_addPilotReferences(cls, pilotRef, ownerGroup, gridType="DIRAC", pilotStampDict={}):
+    def export_addPilotReferences(cls, pilotRef, VO, gridType="DIRAC", pilotStampDict={}):
```

Comment:
Will this be a problem during the update for old pilots to connect to new servers?

Reply:
At the moment this is only called server side (by SiteDirector). There has to be anyway a DB update (I mentioned in the PR description).